lighthouse-core/report/html/report-styles.css

```diff
@@ -648,6 +646,7 @@
 }
 
 .lh-header-sticky {
+  position: -webkit-sticky;
```

Comment:
the time almost certainly isn't now, but do you see a path of us bringing in some build process with prefixers?